format-parquet/src/main/java/org/apache/parquet/avro/AvroSchemaConverter.java

```diff
@@ -54,17 +78,33 @@ public class AvroSchemaConverter {
 
   private final boolean assumeRepeatedIsListElement;
   private final boolean writeOldListStructure;
+  private final boolean writeParquetUUID;
+  private final boolean readInt96AsFixed;
 
   public AvroSchemaConverter() {
-    this.assumeRepeatedIsListElement = ADD_LIST_ELEMENT_RECORDS_DEFAULT;
+    this(ADD_LIST_ELEMENT_RECORDS_DEFAULT);
```

Comment:
This contains a lot of complex logic for the schema conversion, do we have unit tests for this?